docker-compose.yaml

```diff
@@ -31,12 +31,14 @@ services:
       dockerfile: splinterd/Dockerfile-installed-${DISTRO}
       args:
         - REPO_VERSION=${REPO_VERSION}
+        - CARGO_ARGS=${CARGO_ARGS}
     entrypoint: |
       bash -c "
         splinterd \
             --generate-certs \
             -c ./configs/splinterd-node-0-docker.toml \
             --registry-file ./node_registries/nodes.yaml \
+            --db postgres://splinter_admin:splinter_test@splinter-db:5432/splinter \
```

Comment:
This becomes problematic if behind a feature flag. We might need to add a SPLINTERD_ARGS env variable so we can pass this through.

Reply:
added SPLINTERD_ARGS